src/js/angular/rest/connectors.rest.service.js

```diff
@@ -0,0 +1,37 @@
+angular
+    .module('graphdb.framework.rest.connectors.service', [])
+    .factory('ConnectorsRestService', ConnectorsRestService);
+
+ConnectorsRestService.$inject = ['$http'];
+
+const CONNECTORS_ENDPOINT = 'rest/connectors';
+
+function ConnectorsRestService($http) {
+
+    return {
+        getConnectors,
+        initConnector,
+        checkConnectorExists,
+        checkConnector
+    };
+
+    function getConnectors() {
+        return $http.get(CONNECTORS_ENDPOINT);
+    }
+
+    function initConnector(prefix) {
+        return $http.get(`${CONNECTORS_ENDPOINT}/options?prefix=${prefix}`);
+    }
+
+    function checkConnectorExists(prefix) {
```

Comment:
What about `hasConnector`? Also to avoid ambiguity with the next method ?

Reply:
Renamed